Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
```

Comment:
Still trying to understand Gemfile.lcok :-/ why is it modified or included in the first-place…

Reply:
As it is said in the document you pointed out: 
> The next time you run bundle install on the same machine, bundler will see that it already has all of the dependencies you need and skip the installation process. 

It is needed locally to skip the installation process on every start. That's why I wanted to get rid of it, because on every machine can be different versions installed making it problematic to run. On keeping it, we should make sure to update it from time to time, so that we don't carry outdated versions with potential security issues around …

Reply:
Is the change of the Gemfile.lock related anyhow to the use of container-flex addressed in this PR and the corresponding issue? Otherwise, I would propose to have the gemfile update in a separate PR to keep an one-line CSS change separated from a multi-line system-wide package update. 

Could imagine that future generations of code philologists would be grateful ;)

Reply:
> Still trying to understand Gemfile.lcok :-/ why is it modified or included in the first-place…

I put `Gemfile.lock` into `.gitignore` in my Github pages repositories.  This keeps a local copy, and Github creates it as it likes for the GH website.

Reply:
The Gemfile does not specify versions, only packages. The lock file specifies the exact versions of packages to install, and these are, in theory, the tested and "approved" versions that are known to work. 

If you don't care about different people running different versions of the packages, you can remove it from source control. But for larger collaborative projects, ensuring everyone is running the same installed versions can be valuable for not seeing different behaviours across different platforms.

Reply:
The key point, @rettinghaus is this:

> The next time you run bundle install **on the same machine**... 

If it sees you have the gems installed on the same machine, it will skip it. If it does not, it will install it. If you want to force it to rebuild, use:

`bundle install --redownload`

which should force it to reinstall all the Gems.